<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-color:grey;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Stencil</family>
          <pointsize>24</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 8pt &quot;Arial&quot;;
font: 24pt &quot;Stencil&quot;;</string>
        </property>
        <property name="text">
         <string>        Make your account</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>firstName</string>
        </property>
        <property name="buddy">
         <cstring>label_4</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="firstName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>lastName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lastName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>dob</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dob"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>gender</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gender">
        <item>
         <property name="text">
          <string>gander</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>other</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>profilePicture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="profilePicture"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="username"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password"/>
      </item>
      <item>
       <widget class="QPushButton" name="finish">
        <property name="styleSheet">
         <string notr="true">background-color:orange;
border:4px solid black;
border-radius:7px;
font-family:Arial;</string>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
